### PR TITLE
✨ Missing strings

### PR DESCRIPTION
**Pull request by TotallyNotInUse**
I also fixed a string which I thought the translation wasn't as accurately.

### CODE REVIEW

**Comment by TotallyNotInUse** [comment]
> I've noticed that you've included another START_ON_BOOT translation in line 35, line 39 however already has this translation key. Also if you're at it right now already you can introduce the new translation key which was added with #148 recently.

Ok thank you, I fixed it and added the string you suggested.

File: locales/es.json [hunk 1]
```diff
@@ -13,26 +13,32 @@
...
+  "LABEL_LYRICS_NOT_FOUND": "No se ha encontrado letra para esta canción",
   "LABEL_LOADING": "Cargando...",
+  "LABEL_GUEST": "Invitado",
   "LABEL_EXIT": "Salir",
   "LABEL_SETTINGS_TAB_GENERAL": "General",
   "LABEL_SETTINGS_TAB_MACOS": "Mac OS",
   "LABEL_SETTINGS_TAB_MINIPLAYER": "Minireproductor",
   "LABEL_SETTINGS_TAB_SHORTCUTS": "Atajos",
   "LABEL_SETTINGS_TAB_ABOUT": "Acerca de",
+  "LABEL_SETTINGS_TAB_APPEARENCE": "Apariencia",
```
Comment (NNowakowski) [inline locales/es.json:27]: Duplicate of line 42

File: locales/es.json [hunk 2]
```diff
@@ -13,26 +13,32 @@
...
   "LABEL_LOADING": "Cargando...",
+  "LABEL_GUEST": "Invitado",
   "LABEL_EXIT": "Salir",
   "LABEL_SETTINGS_TAB_GENERAL": "General",
   "LABEL_SETTINGS_TAB_MACOS": "Mac OS",
   "LABEL_SETTINGS_TAB_MINIPLAYER": "Minireproductor",
   "LABEL_SETTINGS_TAB_SHORTCUTS": "Atajos",
   "LABEL_SETTINGS_TAB_ABOUT": "Acerca de",
+  "LABEL_SETTINGS_TAB_APPEARENCE": "Apariencia",	
+  "LABEL_SETTINGS_TAB_INTEGRATION": "Integración",
```
Comment (NNowakowski) [inline locales/es.json:28]: Duplicate of line 43

File: locales/es.json [hunk 3]
```diff
@@ -13,26 +13,32 @@
...
   "LABEL_SETTINGS_TAB_GENERAL_KEEP_BACKGROUND": "Mantiene YTMDApp en segundo plano cuando la ventana es cerrada",
   "LABEL_SETTINGS_TAB_GENERAL_SHOW_NOTIFICATIONS": "Mostrar notificaciones",
   "LABEL_SETTINGS_TAB_GENERAL_CONTINUE_WHERE_LEFT_OF": "Continuar desde donde lo has dejado",
   "LABEL_SETTINGS_TAB_GENERAL_DISCORD_RICH_PRESENCE": "Discord Rich Presence",
   "LABEL_SETTINGS_TAB_GENERAL_PAGE_ZOOM": "Zoom de la página",
   "LABEL_SETTINGS_TAB_GENERAL_SELECT_LANGUAGE": "Lenguaje",
+  "LABEL_SETTINGS_TAB_GENERAL_START_ON_BOOT": "Iniciar al encender el equipo (Mac/Win)",
```
Comment (NNowakowski) [inline locales/es.json:35]: Duplicate of line 40

**Comment by TotallyNotInUse** [comment]
> Found a few more duplicates in the file, in the end the file should reach 62 lines. Also please change the target merge branch to ytmdesktop:development

Thank you for all your help, this is my first time collaborating in GitHub so please excuse any mistakes I make. I fixed all your suggested changes and may I ask: why I had to change the branch over to development?

**Comment by NNowakowski** [comment]
> > Found a few more duplicates in the file, in the end the file should reach 62 lines. Also please change the target merge branch to ytmdesktop:development
> 
> Thank you for all your help, this is my first time collaborating in GitHub so please excuse any mistakes I make. I fixed all your suggested changes and may I ask: why I had to change the branch over to development?

Resulting file looks fine now. Can you create a new branch (can name it whatever) on your working copy, beginning from your local development branch (make sure it's on the same state as this projects development branch) and commit the json file on it, then creating another pull request with the base being ytmdesktop:development and the comparison being your newly created branch?

As to why you had to change the branch to development is as to how this repository is designed. As found in https://github.com/ytmdesktop/ytmdesktop/blob/master/CONTRIBUTING.md all development work is done on the development branch with master being pretty much the state of the latest version release. Anything else pending for next release stays in the development branch.

Your local branch was't created with the development branch as a base hence this now bring forth some file conflicts. To clean that up just create a new local branch with development as a base and paste your new code into the json file then proceed as explained above. That will keep the pullrequest nice and clean.

**Comment by TotallyNotInUse** [comment]
Thank you, will do.

**Pull request closed without merging**